### PR TITLE
trim before importing from clipboard

### PR DESCRIPTION
**Pull request by misslivirose**
Fixes https://highfidelity.manuscript.com/f/cases/2264

Test plan: Repeat repro steps in bug, confirm that the entity imports

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8899/app_12934_PR_958d05742208d44bb20b7766f41fb1ada5a63b85.apk

File: scripts/system/edit.js [hunk 1]
```diff
@@ -1679,7 +1679,7 @@ function importSVO(importURL) {
         visible: true
     });
 
-    var success = Clipboard.importEntities(importURL);
+    var success = Clipboard.importEntities(importURL.trim());
```
Comment (sethalves) [inline scripts/system/edit.js]: Could this be fixed in Octree::readFromURL or getMarketplaceID ?  I suspect that if we fix this in this way, we'll be fixing it again, later.

Comment (misslivirose) [reply scripts/system/edit.js]: Yeah, good call.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8899/HighFidelity-Beta-PR12934-958d05742208d44bb20b7766f41fb1ada5a63b85.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8899/HighFidelity-Beta-Interface-PR12934-958d05742208d44bb20b7766f41fb1ada5a63b85.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8899/HighFidelity-Beta-PR12934-958d05742208d44bb20b7766f41fb1ada5a63b85.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8899/HighFidelity-Beta-Interface-PR12934-958d05742208d44bb20b7766f41fb1ada5a63b85.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8905/app_12934_PR_d2335ea9f6e3675e28ba86b7e3957fc2c72664ac.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8905/HighFidelity-Beta-PR12934-d2335ea9f6e3675e28ba86b7e3957fc2c72664ac.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8905/HighFidelity-Beta-Interface-PR12934-d2335ea9f6e3675e28ba86b7e3957fc2c72664ac.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8905/HighFidelity-Beta-PR12934-d2335ea9f6e3675e28ba86b7e3957fc2c72664ac.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8905/HighFidelity-Beta-Interface-PR12934-d2335ea9f6e3675e28ba86b7e3957fc2c72664ac.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Even with space before and after the URL, the model was still imported.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9185/app_12934_PR_afea18678496ba83b0d0ea49b6c3f0d4d0539bef.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9185/HighFidelity-Beta-PR12934-afea18678496ba83b0d0ea49b6c3f0d4d0539bef.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9185/HighFidelity-Beta-Interface-PR12934-afea18678496ba83b0d0ea49b6c3f0d4d0539bef.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9185/HighFidelity-Beta-PR12934-afea18678496ba83b0d0ea49b6c3f0d4d0539bef.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9185/HighFidelity-Beta-Interface-PR12934-afea18678496ba83b0d0ea49b6c3f0d4d0539bef.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
With a space before and after the URL the link will still load. Tested with an absurd amount of spaces and the model did fail to load. However that would be a big user error (edge case)

**Pull request merged**